src/components/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  children: React.ReactNode;
+};
+
+const GENDER_FEMALE = 'f';
+
+export const PersonLink: React.FC<Props> = ({ person, children }) => {
+  const { slug } = person;
+  const to = `../people/${slug}`;
+  const className = person.sex === GENDER_FEMALE ? 'has-text-danger' : '';
+
+  return (
+    <Link to={to} className={className}>
+      {children}
```

Comment:
```suggestion
  person: Person;
};

const GENDER_FEMALE = 'f';

export const PersonLink: React.FC<Props> = ({ person }) => {
  const { slug, name, sex } = person;
  const to = `../people/${slug}`;
  const className = sex === GENDER_FEMALE ? 'has-text-danger' : '';

  return (
    <Link to={to} className={className}>
      {name}
```
In this task you can do it without children